### ISSUE TITLE
[Incident] CarbonPlan Hub inaccessible due to expired cert

### ISSUE THREAD

**Issue by maxrjones** [2i2c-org/infrastructure#3274]
### Summary

The CarbonPlan Hub is down due to an expired certificate

### Impact on users

This is urgent because users cannot access the Hub.

### Important information

- Hub URL: https://carbonplan.2i2c.cloud/
- Support ticket ref: {{ INSERT SUPPORT REF HERE }}

### Tasks and updates

- [ ] Discuss and address incident, leaving comments below with updates
- [ ] Incident has been dealt with or is over
- [ ] Copy/paste the after-action report below and fill in relevant sections
- [ ] Incident title is discoverable and accurate
- [ ] All actionable items in report have linked GitHub Issues

<details>
<summary>After-action report template</summary>

```
# After-action report

These sections should be filled out once we've resolved the incident and know what happened.
They should focus on the knowledge we've gained and any improvements we should take.

## Timeline

_A short list of dates / times and major updates, with links to relevant comments in the issue for more context._

All times in {{ most convenient timezone}}.

- {{ yyyy-mm-dd }} - [Summary of first update](link to comment)
- {{ yyyy-mm-dd }} - [Summary of another update](link to comment)
- {{ yyyy-mm-dd }} - [Summary of final update](link to comment)


## What went wrong

_Things that could have gone better. Ideally these should result in concrete
action items that have GitHub issues created for them and linked to under
Action items._

- Thing one
- Thing two

## Where we got lucky

_These are good things that happened to us but not because we had planned for them._

- Thing one
- Thing two

## Follow-up actions

_Every action item should have a GitHub issue (even a small skeleton of one) attached to it, so these do not get forgotten. These issues don't have to be in `infrastructure/`, they can be in other repositories._

### Process improvements

1. {{ summary }} [link to github issue]
2. {{ summary }} [link to github issue]

### Documentation improvements

1. {{ summary }} [link to github issue]
2. {{ summary }} [link to github issue]

### Technical improvements

1. {{ summary }} [link to github issue]
2. {{ summary }} [link to github issue]
```

</details>

**Comment by yuvipanda** [comment]
This was sorted out and I responded on freshdesk.

**Comment by yuvipanda** [comment]
(Treating this as a light-weight incident report)

This was caused by a `404` in the let's encrypt process, causing the certificate renewal to get stuck for the last 30 days.

here's the relevant part of `k -n prod describe certificate`

```
  Warning  Failed  2m36s  cert-manager-certificates-issuing  The certificate request has failed to complete and will be retried: Failed to wait for order resource "https-auto-tls-zlxhr-4134250225" to become ready: order is in "errored" state: Failed to retrieve Order resource: 404 urn:ietf:params:acme:error:malformed: No order for ID 208417452356
```

I deleted the certificate and restarted everything in `cert-manager` namespace, which fixed it.

The fundamental issues here are:

1. There was no alerts for us that a cert was about to expire. I opened https://github.com/2i2c-org/infrastructure/issues/3278 to track
2. There was not enough engineering coverage to tackle this for a few hours. We're currently in the process of hiring another engineer, which should improve response times (https://2i2c.org/jobs/2023/23qq4-open-source-infrastructure-engineer/)
